README.md

```diff
@@ -20,15 +24,20 @@ In order to check SSO using SAML2, please follow these steps
    For the service provider, configure SAML2 Web SSO under Inbound Authentication Configuration. In this configuration,
    use following parameters and options,
      
-       Issuer - sampleApp  
+       Issuer - SampleApp  
        Assertion Consumer URLs - http://localhost:8080/SampleApp/home.jsp 
+       Enable Attribute Profile - True
+       Include Attributes in the Response Always - True
 
 
    Keep other default settings as it is and save the configuration.
    
-   Next, expand the Claim Configuration section. In this configuration, add the claims you need to retrive from the
-    web app.
-
+   Next, expand the Claim Configuration section. In this configuration, Set the following config and add the claims you 
+   need to retrieve (ex: http://wso2.org/claims/username) from the web app.
```

Comment:
Shall we add a screenshot here of the claim configs to give more context?

Reply:
Resolved in https://github.com/wso2-extensions/identity-agent-sso/pull/74/commits/40cf2ea6c95a8f59b51e9cb4f053c00b21a4ea92